orcestra_book/reports/HALO-20240921a.md

```diff
@@ -0,0 +1,240 @@
+Crew before take-off.
+```
+
+```{card}
+:img-top: ../figures/HALO-20240921a/IMG_5077_jellyfishanvil_1312UTC.jpeg
+Inverted jellyfish like anvil remnants at 13:12 UTC.
+```
+
+```{card}
+:img-top: ../figures/HALO-20240921a/IMG_5084_turtleinCu_1339UTC.jpeg
+Shallow cumulus cloud fields at 13:39 UTC, with the larger aggregate in the center resembling a turtle.
+```
+
+```{card}
+:img-top: ../figures/HALO-20240921a/DSC_9230_Cbwithpileus_circeast_1413UTC.JPG
+Approx. 15 km deep Cb tower with pileus cloud shot out of the cockpit during our deviation into the eastern circle  (14:13 UTC).
+```
+
+````
+
+
+````{card-carousel} 2
+
+```{card}
+:img-top: ../figures/HALO-20240921a/IMG_5125_thincloudarcs_circeast_1446UTC.jpeg
+Sheared raining clouds feeding several cloud holes, but apparently producing very weak gust fronts that trigger only very shallow cloud arcs. A lot of mid-level outflow and a deep convective system in the background (14:46 UTC).
+```
+
+```{card}
+:img-top: ../figures/HALO-20240921a/IMG_5135_mixCbaltocu_circeast_1505UTC.jpeg
+Mix of altocumulus, congestus and cumulonimbus clouds in the second round of the eastern circle (15:05 UTC).
+```
+
+```{card}
+:img-top: ../figures/HALO-20240921a/DSC_9394_cloudmirrors_ECtrack_1750UTC.jpeg
+Clouds mirrored in the ocean after the EC underpass at 17:50 UTC.
+```
+
+```{card}
+:img-top: ../figures/HALO-20240921a/IMG_5254_cumulustowers_circmeteor2_1929UTC.jpeg
+Five cumulus towers after leaving the second Meteor circle (19:29 UTC).
+```
+
+````
+
+## Instrument status & quicklooks
+
+```{instrument-table}
+```
+````{card-carousel} 2
+
+```{card}
+:img-top: ../figures/HALO-20240921a/BACARDI_QL_20240921a.png
+BACARDI
+```
+
+```{card}
+:img-top: ../figures/HALO-20240921a/20240921_circle_profiles.png
+Dropsondes
+```
+
+```{card}
+:img-top: ../figures/HALO-20240921a/hamp_fullflight_HALO-20240921a.png
+HAMP
+```
+
+```{card}
+:img-top: ../figures/HALO-20240921a/hamp_ec_under_HALO-20240921a.png
+HAMP during EarthCARE underpass
+```
+
+```{card}
+:img-top: ../figures/HALO-20240921a/hamp_radar_ec_under_HALO-20240921a.png
+Radar during EarthCARE underpass
+```
+
+<!-- ```{card}
+:img-top: ../figures/HALO-20240921a/HALO_20240921a_KT19_QL.png
+KT19 Timeseries of brigthness temperature.
+```
+
+```{card}
+:img-top: ../figures/HALO-20240921a/HALO_20240921a_SMART_QL.png
+SMART Time series of upward and downward spectral irradiadiance, and albedo.
+``` -->
+
+```{card}
+:img-top: ../figures/HALO-20240921a/specmacs.png
+SpecMACS.
+```
+
+<!-- ```{card}
+%:img-top: ../figures/HALO-20240921a/QL_VELOX_HALO_20240921a.jpg
+VELOX broadband channel with EarthCARE underpass
+``` -->
+
+```{card}
+:img-top: ../figures/HALO-20240921a/HALO-20240921-wales-ec.png
+WALES (backscatter, polarization and humidity) at EC overpass
+```
+
+<!-- ```{card}
+:img-top: ../figures/HALO-20240921a/0921-wales-3d.jpg
+WALES (3D backscatter)
+```
+
+```{card}
+:img-top: ../figures/HALO-20240921a/0921-wales-x.png
+WALES (cross section)
+``` -->
```

Comment:
The CI crashes in line 205 with the message: "All children of a 'card-carousel' should be 'card' [design.card]". It seems like you wanted to comment out the not yet available QL images in html syntax which did not work within the carousel. I'd suggest to either comment out using `%` at the beginning of respective lines or delete the not yet available parts.